rust/processor/src/processors/default_processor.rs

```diff
@@ -297,7 +304,7 @@ pub fn process_transactions(
     flags: TableFlags,
```

Comment:
nit: should we also move "use the TableFlags to clear models" part out of this function?

Reply:
yeah makes sense, will move to upper level